editor/utils/graphql.ts

```diff
@@ -47,6 +47,13 @@ const fetcher = <T>([query, variables, token, hasuraRoleName]: [
       Authorization: `Bearer ${token}`,
       "x-hasura-role": hasuraRoleName,
     },
+    // Explicitly set cache behavior for Next.js 15
+    fetch: (url, options) => {
+      return fetch(url, {
+        ...options,
+        cache: "no-store",
```

Comment:
Next.js 15 changed the default `fetch()` cache behavior from `cache: 'force-cache'` to `cache: 'no-store'`. Added explicit cache configuration to GraphQL requests to ensure consistent behavior.